### PR TITLE
Sh jl/lammps

### PR DESCRIPTION
**Pull request by Stefaanhess**
**LAMMPS Interface**

- added JIT support for spk-models
- spkdeploy script to create JIT models
- C++ interface for LAMMPS
- script for patching C++ interface into lammps installation
- tutorial for installation and usage of LAMMPS interface
- example files and trained model to test lammps interface

### CODE REVIEW

**Review by mgastegger — APPROVED** [review]

**Review by mgastegger — APPROVED** [review]

**Comment by Stefaanhess** [comment]
should be ready for merge now

**Review by mgastegger — CHANGES_REQUESTED** [review]
Looks good, there is just a small typo in Tensosboard in the README.md

File: README.md [hunk 1]
```diff
@@ -25,6 +20,7 @@ extensible to custom atomistic architectures.
 - Atomic Simulation Environment (ASE) 3.21
 - NumPy
 - PyTorch 1.9
+- PyTorch Lightning 1.4.5
```
Comment (NiklasGebauer) [inline README.md:23]: Versions should match setup.py (1.9)

**Review by NiklasGebauer — CHANGES_REQUESTED** [review]
We should update the version numbers from requirements in the readme (or maybe remove them as they change all the time?)

**Review by NiklasGebauer — APPROVED** [review]
Looking fine I think.

**Review by mgastegger — APPROVED** [review]

**Pull request merged**